Reword builtin-library.md a little bit. (#129)
langium/langium-website: hugo/content/guides/builtin-library.md

However, when trying to navigate to the builtin library elements, vscode will show users an error message, complaining that it cannot find the builtin library file.
This is expected, as the builtin library only lives in memory.

> To fix this issue, we need to implement a custom `FileSystemProvider` on the client(`src/extension.ts` in the *hello world* example) that allows navigation to the builtin library files:

Reviewer: ```suggestion
To fix this issue, we need to implement a custom `FileSystemProvider` for the vscode extension(`src/extension.ts` in the *hello world* example) that allows navigation to the builtin library files:
```

            }));
    }

    stat(uri: vscode.Uri): vscode.FileStat {
        const date = Date.now();
        return {
            ctime: date,
            mtime: date,

>             size: library.length, // Library is undefined!

Reviewer: ```suggestion
            size: builtinHelloWorld.length,
```
